### PR TITLE
User will not scroll out of DimagiSphere

### PR DESCRIPTION
**Pull request by rickypchen**
@esoergel @czue 
Setting maxBounds and zoom limits for DimagiSphere so that user will not drag the map out of bounds

### CODE REVIEW

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 1]
```diff
@@ -146,13 +146,25 @@ var projectMapInit = function(mapboxAccessToken) {
...
-        maxZoom: 10,
+        maxZoom: 6,
+        minZoom: 2,
         id: mapId,
-        accessToken: mapboxAccessToken
+        accessToken: mapboxAccessToken,
+        noWrap: true,
     }).addTo(map);
 
+    var southWest = L.latLng(-85.0, -180.0),
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: 'L' is not defined. (no-undef)

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 2]
```diff
@@ -146,13 +146,25 @@ var projectMapInit = function(mapboxAccessToken) {
...
+        northEast = L.latLng(85.0, 180.0),
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: 'L' is not defined. (no-undef)

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 3]
```diff
@@ -146,13 +146,25 @@ var projectMapInit = function(mapboxAccessToken) {
...
+        bounds = L.latLngBounds(southWest, northEast);
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: 'L' is not defined. (no-undef)

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 4]
```diff
@@ -146,13 +146,25 @@ var projectMapInit = function(mapboxAccessToken) {
...
+
+    map.setMaxBounds(bounds);
+    map.on('drag', function(){
+        map.panInsideBounds(bounds, {animate:false});
+    })
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: Missing semicolon. (semi)

File: .eslintrc.js [hunk 5]
```diff
@@ -19,6 +19,7 @@ module.exports = {
         "ko": false,
         "_": false,
         "ga_track_event": false,
+        "L": true
```
Comment (esoergel) [inline .eslintrc.js]: nit: we usually use trailing commas in javascript object literals so git blame is easier.  I'm actually surprised dimagimon didn't complain.

**Pull request merged**